podium/talks/urls.py

```diff
@@ -4,4 +4,5 @@
 
 urlpatterns = [
     url(r'^submit/', views.submit_talk_view, name='talks-submit'),
+    url(r'^sessions/', views.session_list_view, name = 'sessions'),
```

Comment:
It's good practice in Django to prepend your alias name for a view with the name of the app it's included in, to avoid name collisions with other apps that may use the same name for a view.